circle.yml

```diff
@@ -173,6 +231,24 @@ jobs:
             echo $name
             ghr -u $CIRCLE_PROJECT_USERNAME -r $CIRCLE_PROJECT_REPONAME -n "$name" $prerelease_flag $CIRCLE_TAG ~/package
 
+  consensus-tests:
+    environment:
+      TESTS_REV: v7.0.0
+    executor: linux-gcc-latest
+    steps:
+      - build_silkworm
+      - attach_workspace:
+          at: ~/package
+      - run:
+          name: "Unpack evmone"
+          background: true
```

Comment:
So this runs parallel? Does it really take that much time to risk any potential wrong order execution here?